genesis/beacons.go

```diff
@@ -11,56 +11,9 @@ import (
 // getIPs returns the beacon IPs for each network
 func getIPs(networkID uint32) []string {
 	switch networkID {
-	case constants.MainnetID:
+	case constants.ColumbusID:
 		return []string{
-			"54.94.43.49:9651",
-			"52.79.47.77:9651",
-			"18.229.206.191:9651",
-			"3.34.221.73:9651",
-			"13.244.155.170:9651",
-			"13.244.47.224:9651",
-			"122.248.200.212:9651",
-			"52.30.9.211:9651",
-			"122.248.199.127:9651",
-			"18.202.190.40:9651",
-			"15.206.182.45:9651",
-			"15.207.11.193:9651",
-			"44.226.118.72:9651",
-			"54.185.87.50:9651",
-			"18.158.15.12:9651",
-			"3.21.38.33:9651",
-			"54.93.182.129:9651",
-			"3.128.138.36:9651",
-			"3.104.107.241:9651",
-			"3.106.25.139:9651",
-			"18.162.129.129:9651",
-			"18.162.161.230:9651",
-			"52.47.181.114:9651",
-			"15.188.9.42:9651",
-		}
-	case constants.FujiID:
-		return []string{
-			"3.214.61.227:9651",
-			"52.206.218.4:9651",
-			"44.194.128.146:9651",
-			"3.143.146.90:9651",
-			"3.142.66.84:9651",
-			"3.142.32.15:9651",
-			"44.240.251.247:9651",
-			"44.224.22.217:9651",
-			"52.13.58.52:9651",
-			"18.163.142.196:9651",
-			"16.162.54.143:9651",
-			"18.167.153.71:9651",
-			"52.29.183.160:9651",
-			"18.159.63.226:9651",
-			"3.65.152.247:9651",
-			"34.247.100.96:9651",
-			"34.250.89.215:9651",
-			"54.228.143.65:9651",
-			"54.232.253.20:9651",
-			"54.94.159.80:9651",
-			"54.94.242.98:9651",
+			"104.154.245.81:9651",
```

Comment:
Would it be possible to add a Hostname instead of a IP here or is DNS resolution not possible in this case?
I'm just asking because the IPs *might* change in the future and it would be more future-proof to have specific service-DNS entries for that.